docs/usage/polling.md

```diff
@@ -1,16 +1,15 @@
 # Polling Legacy Z-Wave Devices
 
-Some legacy devices don't report all of their values automatically and require polling to automatically update their values. Some platforms, such as Smartthigns and Open Z-Wave, poll such devices automatically whether you need them polled or not. Because such polling can have detrimental effects on your mesh network, zwave-js does not poll devices by default. Polling can quickly lead to network congestion and should be used very sparingly and only where necessary.
+Some legacy devices don't report all of their values automatically and require polling to automatically update their values. Some platforms, such as Smartthigns and Open Z-Wave, poll such devices automatically whether you need them polled or not. Because such polling can have detrimental effects on your mesh network, Z-Wave JS does not poll devices by default. Polling can quickly lead to network congestion and should be used very sparingly and only where necessary.
```

Comment:
```diff
- Smartthigns
+ Smartthings
```

Reply:
```diff
- Open Z-Wave
+ OpenZWave
```

Reply:
Maybe just remove the references to the other platforms, because OZW does not poll devices by default either.